### PR TITLE
Fix typo on ENGINE.md

### PR DESCRIPTION
**Pull request by filfreire**
"build in" to "built-in"

/CC @Gh0stBlade @TeslaRus

### CODE REVIEW

**Comment by Gh0stBlade** [comment]
There's many more errors which need correcting.

**Comment by TeslaRus** [comment]
yep, so wait more fixes in .md

**Comment by filfreire** [comment]
@Gh0stBlade @TeslaRus I fixed multiple typos and errors now.
Kindly CR again.

File: ENGINE.md [hunk 1]
```diff
@@ -41,23 +41,23 @@
...
-    - `trigger` - here is main (in game) sector trigger handler / parser, objects functions caller;
-    - `world` - main level data base storage (exclude sound): models, entities, rooms, meshes a.t.c. here are level loader / destructor and interface for accessing to rooms / entities by coordinates / id's;
+    - `skeletal_model` - contains base model animation representation structures, and in-game usage unique skeletal model structure; implemented smoothed skeletal model update algorithm, multi animation system algorithm and multi-targeting bone mutators algorithm (head tracking, weapons targeting);
+    - `trigger` - here is main (in-game) sector trigger handler/parser, objects functions caller;
+    - `world` - main level database storage (exclude sound): models, entities, rooms, meshes a.t.c. here are level loader/destructor and interface for accessing to rooms/entities by coordinates/ids;
```
Comment (Gh0stBlade) [inline ENGINE.md]: excluding

Comment (filfreire) [reply ENGINE.md]: @Gh0stBlade improved

**Comment by Gh0stBlade** [comment]
I will finish the rest.

**Pull request merged**